bazel/repository_locations.bzl

```diff
@@ -1119,8 +1119,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WebAssembly for Proxies (C++ host implementation)",
         project_desc = "WebAssembly for Proxies (C++ host implementation)",
         project_url = "https://github.com/proxy-wasm/proxy-wasm-cpp-host",
-        version = "2cf68d6c00b4141ecf2afb5e5d51ebd21a27ed43",
-        sha256 = "6314d3e36e8aa9c4e792772b66f773fcfd58affcd4fd437c6d3dc540165e7fc6",
+        version = "4ddbed3c8c8c1aa76db8157e22ae3be676e76ac1",
+        sha256 = "e9c05ed5e827e256f4725735c2ed3d5420fc95f5bce052b42f93aa37248a1f42",
```

Comment:
You need to update the `release_date` below. I'm not sure why the format check didn't catch it.

Reply:
Done.